### PR TITLE
Docs: export async deprecated

### PR DESCRIPTION
**Pull request by TorsteinHonsi**
Article on how to migrate away from export `async`. Not to be listed in the sidebar, only to be referenced from the other repo and from announcements.

### CODE REVIEW

File: docs/export-module/deprecated-async-option.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
+Deprecating the export server async option
+------------------------------------------
+
+Version 2.2 of the Highcharts Node.js Export Server, due December 1st 2021, will deprecate the `async` option. The reasons for that is that it makes the application dependent on saving state, and complicates load balancing. Furthermore it is no longer needed in browsers because we can render the returned image directly as a base64 encoded `src`.
+
+Here's how to migrate from the `async` option.
+
+### Browser
+
+In the browser, the config can be posted and the returns be rendered using the FormData, fetch and FileReader API's. See the [example on jsFiddle](https://jsfiddle.net/highcharts/mt2s95Ln/) for how to set this up with an async-await based syntax.
```
Comment (goransle) [inline docs/export-module/deprecated-async-option.md:10]: Maybe "the returned data" in place of "the returns"?

File: docs/export-module/deprecated-async-option.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+If legacy browsers are are a requirement, see also the [old jQuery based demo](https://jsfiddle.net/highcharts/edyucqkw/). Note that this approach uses the export server's `b64` option to do the base64 encoding on the server side.
```
Comment (goransle) [inline docs/export-module/deprecated-async-option.md:12]: "are are"

File: docs/export-module/deprecated-async-option.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+### Node
+
+If you're running a node environment, the recommended approach would be to use the export server as a node module instead of posting content to Highsoft's online server. For details, see the chapter [Using as a Node.js Module](https://github.com/highcharts/node-export-server/blob/master/README.md#using-as-a-nodejs-module) in the project repo.
```
Comment (goransle) [inline docs/export-module/deprecated-async-option.md:16]: capitalize Node? Or maybe Node.js?

**Review by goransle — APPROVED** [review]
Looks good, only a few grammar nitpicks 🤓

File: docs/export-module/deprecated-async-option.md (hunk 3 above)
Comment (TorsteinHonsi) [reply docs/export-module/deprecated-async-option.md:16]: Thanks! Node.js is the trademark they're using.

**Pull request merged**